CHANGELOG.rst

```diff
@@ -52,6 +52,8 @@ Changed
 - Differenciate "flash alerts" notifications (#594)
 - Improve menu bar spacing, put history and settings in a submenu (#739)
 - Change Dockerfile to install python dependencies at build time (#793)
+- Project code is not required when updating other project settings (#774)
```

Comment:
Is this the same as "Ask for private code to delete project or project history (#796)" above?

Reply:
Ah, now I recall, it's different.

Maybe improve it to "Updating the private code is no longer required when editing other project settings"?

Reply:
Is it better?

Reply:
Yes, thanks!